phpBB/phpbb/install/converter/factory/dbal_connection_destination_factory.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
We cant actually. This file is essential for our saved data from install_Config to create DBAL driver manager object.